heroku.env

```diff
@@ -2,4 +2,4 @@
 HOST=0.0.0.0
 NODE_ENV=production
 NPM_CONFIG_PRODUCTION=false
-
+AXIOS_BASE_URL=http://localhost:3000
```

Comment:
This actually should not go into `heroku.env `, it should go to `template.env`...